dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Insert some test data in the db
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+# Build, run and execute the pg_basebackup action
+docker container run --rm -i --name pg-backup --network=host $(docker build --build-arg USER_ID=$(id -u) -f dev_tools/Dockerfile-backup -q -t backup .) /bin/sda-backup --action pg_basebackup
```

Comment:
There should no be a need for the UID

Reply:
There were several problems when I was not using it, e.g: you cannot delete the folder after the testing was done and there were some others which I don' remember now since it was done in May. But I can say that it is needed

Reply:
That's because you are writing to local disc.

If you use docker volumes for all read/write operations you will not have to do any local cleanup afterwards, just remember to add `-v` when you take down the compose environment so the volumes gets deleted.

If the container runs as UID `70` instead of `65354` you don't need to do any `chown` operations either as the unpacked files have the correct owner.